bio_files_processor.py

```diff
@@ -0,0 +1,33 @@
+def convert_multiline_fasta_to_oneline(input_fasta, output_fasta: str = "output.fasta"):
+    """
+    Конвертирует многострочный FASTA-файл в однострочный для каждой последовательности.
+    
+    Параметры:
+    - input_fasta: Путь к входному FASTA-файлу.
+    - output_fasta: Путь к выходному FASTA-файлу, куда будет записан результат.
+                      Если не указан, выводит результат на экран.
+    """
+
+    if not output_fasta:
+        output_fasta = "output.fasta"  
+    
+    resulting_sequences = []
+    with open(input_fasta, "r") as fasta:
+        one_line_sequence = []
+        for line in fasta:
+            line = line.strip()
+            if line[0]==">":  
```

Comment:
Разницы 0, но я бы написал: 
```suggestion
            if line.startswith(">"):  
```
так боле питонисто и по-английски получается